main.go

```diff
@@ -1,18 +1,36 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
 	"log"
 	"net/http"
+	"os"
 
 	// 3rd party router package
 	"github.com/google/jsonapi"
+	"github.com/gorilla/handlers"
 	"github.com/gorilla/mux"
+	"github.com/thanhpk/randstr"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/endpoints"
 )
 
 type jobTypeEnum string
 type experienceEnum string
 
+// Move this somewhere more appropriate
+var (
+	authConfig = &oauth2.Config{
+		RedirectURL:  os.Getenv("ORIGIN") + "/auth/callback",
+		ClientID:     os.Getenv("SLACK_CLIENT_ID"),
+		ClientSecret: os.Getenv("SLACK_CLIENT_SECRET"),
+		Scopes:       []string{"identity.basic"},
+		Endpoint:     endpoints.Slack,
+	}
```

Comment:
nice! is this working with slack?

Reply:
Yes, but just the log in part.  It's not storing in a session or anything...